service-workers/service-worker/resources/fetch-event-test-worker.js

```diff
@@ -155,6 +155,12 @@ function handleIsHistoryNavigation(event) {
   event.respondWith(new Response(body));
 }
 
+function handleUseAndIgnore(event) {
+  const request = event.request;
+  request.text();
```

Comment:
I think we should assert the body value here as well or in a different test. Otherwise it's still unclear whether the server worker can read it.

Reply:
This is tested elsewhere: see "FetchEvent#body is a ReadableStream" in fetch-event.https.html.